addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -259,6 +259,8 @@ def _compute_number_of_days(self):
             if allocation_unit != 'hour':
                 allocation.number_of_days = allocation.number_of_days_display
             else:
+                if not allocation.employee_id._get_hours_per_day(allocation.date_from):
+                    raise ValidationError(_("Set a valid working schedule for employee %(employee)s to enable hour allocation.", employee=allocation.employee_id.name))
```

Comment:
When you add a translatable string in a stable version, you have to update the translation file too.